dbms/src/Server/ServerInfo.h

```diff
@@ -38,7 +37,7 @@ class ServerInfo
         /// number of logical CPU cores
         UInt16 logical_cores = std::thread::hardware_concurrency();
         /// number of physical CPU cores
-        UInt16 physical_cores = getNumberOfPhysicalCPUCores();
+        UInt16 physical_cores = std::thread::hardware_concurrency();
```

Comment:
actually `std::thread::hardware_concurrency();` get the total logical cpu cores, maybe set `UInt16 physical_cores = std::thread::hardware_concurrency() / 2;`